src/index.js

```diff
@@ -132,6 +156,27 @@ class HawkWebpackPlugin {
     return maps;
   }
 
+  /**
+   * Find commits in the project directory
+   * @return {Commit[]}
+   */
+  findCommits() {
+    const options = {
+      repo: this.commits.repo,
+      number: this.commits.number,
+      fields: ["hash", "subject", "authorEmail", "authorDate"],
+    }
+
+    const commits = gitlog(options);
+
+    return commits.map(commit => ({
+      hash: commit.hash,
+      title: commit.subject,
+      author: commit.authorEmail,
```

Comment:
Can we parse author's name here? And send in author field `Ilya Moroz <email@email.com>`